src/EnergyPlus/OutputProcessor.hh

```diff
@@ -870,8 +870,8 @@ void SetupOutputVariable(EnergyPlusData &state,
                          OutputProcessor::SOVTimeStepType TimeStepTypeKey, // Zone, HeatBalance=1, HVAC, System, Plant=2
                          OutputProcessor::SOVStoreType VariableTypeKey,    // State, Average=1, NonState, Sum=2
                          std::string_view const KeyedValue,                // Associated Key for this variable
-                         ObjexxFCL::Optional_string_const ReportFreq = _,  // Internal use -- causes reporting at this freqency
-                         ObjexxFCL::Optional_int_const indexGroupKey = _   // Group identifier for SQL output
+                         std::string_view const ReportFreq = {},           // Internal use -- causes reporting at this freqency
+                         int const indexGroupKey = -999                    // Group identifier for SQL output
```

Comment:
OIC, the -999 isn't related to the default numeric for autosize.  I guess -999 is fine as a sentinel, but I'm sure there's a bit more cleaning that could be done on a future PR.